src/main/java/seedu/address/model/person/Contact.java

```diff
@@ -91,6 +94,25 @@ public boolean isSameContact(Contact otherContact) {
                 && otherContact.getName().equals(getName());
     }
 
+    /**
+     * Gives the array of contacts that are linked under this contact.
+     */
+    public Contact[] getChildren() {
+        // default return value
+        // TODO add to DG
+        return new Contact[]{};
+    }
```

Comment:
These haven't been implemented by `Organization` yet, but we can do them later. Perhaps we should explicitly override and add a `TODO` in the `Organization` class as well.

Reply:
I've added a todo comment in the organization class for future development